docs/datadog_ktx_module.md

```diff
@@ -0,0 +1,49 @@
+# Datadog Kotlin Extensions package
+
+Provides useful Kotlin extensions on top of our core `dd-sdk-android` SDK.
+
+**Note**: This package is just an extension of our main package so you should add both dependencies into your gradle file.
+
+## Setup
+```conf
+repositories {
+    maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+}
+
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android:x.x.x"
+    implementation "com.datadoghq:dd-sdk-android-ktx:x.x.x"
+}
+```
+
+### Extensions
+
+1. Span extension methods:
+
+```kotlin
+
+span.setError(throwable)
+
+// or for setting a message
+span.setError(message)
+
+```
+
+```kotlin
+
+// will automatically create a span around the provided lambda
+withinSpan(spanName, parentSpan){
+  // your code here
+}
+
+```
+
+2. Extensions for OkHttp request builder:
+
+```kotlin
+
+  // if you are using our TracingInterceptor for OkHttp request it will automatically add the provided parentSpan as the parent
+  // of the span we automatically create around the request.
```

Comment:
It reads a bit convoluted, how about : 
if you are using our TracingInterceptor for OkHttp request, it will add the provided parentSpan as the parent of the span created around the request.

Reply:
Yup you are right...will change that